package.json

```diff
@@ -17,7 +17,7 @@
   },
   "homepage": "https://github.com/thomotron/sfp-donation-portal#readme",
   "dependencies": {
-    "better-sqlite3": "^5.4.3",
```

Comment:
Why the dependency upgrade?

Reply:
This allows the project to run on NodeJS x.16 and above, which saves headaches for building the image and in general

Reply:
Fair!